<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastrar</class>
 <widget class="QWidget" name="Cadastrar">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cadastrar</string>
  </property>
  <widget class="QLineEdit" name="campo_nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="campo_CPF">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="campo_RG">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>RG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_concluir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>concluir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_cancelar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Casa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_casa">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>campo_nome</tabstop>
  <tabstop>campo_CPF</tabstop>
  <tabstop>campo_RG</tabstop>
  <tabstop>comboBox_casa</tabstop>
  <tabstop>botao_concluir</tabstop>
  <tabstop>botao_cancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
